recipe/meta.yaml

```diff
@@ -11,11 +11,11 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   {{ hash_type }}: {{ hash_val }}
-  # patches:
-  #   - python35-compat.patch  # [py35]
+  patches:
+     - python35-compat.patch  # [py35]
```

Comment:
Have you made this patch available upstream? I'm sure they would love to know about it! Providing the link as a comment will also allow us to cross-reference when we upgrade the recipe to determine if the patch is no longer needed.